test/trento/domain/sap_system/sap_system_test.exs

```diff
@@ -4202,6 +4210,365 @@ defmodule Trento.SapSystemTest do
     end
   end
 
+  describe "instance marked absent/present" do
+    test "should mark as absent a previously registered database instance" do
```

Comment:
I'm not sure if mixing 2 different flows is a good idea, as it "hides" them.
I'm talking about the scenario where you test here that an already marked instance is not re-marked.
No need for a change, the test is totally correct.
For the future